site/docs/components/component_overview.mdx

```diff
@@ -63,7 +63,7 @@ Component                                                        | Status
 Dropdown                                                         | <StatusLabel type="error">Deprecated</StatusLabel>   |                                                                               |
 [Footer](/components/footer)                                     | <StatusLabel type="alert">Pre-release</StatusLabel>  |                                                                               | [<IconCheck/>](/storybook/react/?path=/story/components-footer--default)
```

Comment:
Footer is marked as stable in the component docs?

Reply:
Good catch. This was the one that we kept pondering if it's stable or not so of course it got half way there :D I'll fix it.

Reply:
Stable or not? 🤔 
Is it time to toss a coin?